distros/kubernetes/nvsentinel/charts/syslog-health-monitor/templates/_helpers.tpl

```diff
@@ -60,3 +60,198 @@ Create the name of the service account to use
 {{- default "default" .Values.serviceAccount.name }}
 {{- end }}
 {{- end }}
+
+{{/*
```

Comment:
is this no longer needed -- https://github.com/NVIDIA/NVSentinel/blob/main/distros/kubernetes/nvsentinel/charts/syslog-health-monitor/templates/configmap.yaml#L40  ? Can we drop it?